testing/integration/test_ot.py

```diff
@@ -15,6 +16,7 @@ def test_stack_connectivity(self):
         logger.debug('Running test_stack_connectivity')
         self._clean_stack()
         self._setup_stack()
+        time.sleep(15)
```

Comment:
What's this for specific to this case?  If it's something we'd need always then I'd put it in _setup_stack()

Reply:
Done.